src/Uno.UI/UI/Xaml/Window.Android.cs

```diff
@@ -26,8 +30,29 @@ public Window()
 
 		private void InternalSetContent(UIElement value)
 		{
-			_content = value;
-			ApplicationActivity.Instance?.SetContentView(value);
+			if (_main == null)
```

Comment:
I prefer this approach, but did you test what happens when you restart the activity? (Ie hit back button to back out of the app, and restore it by pressing the icon.) I think you may need to adjust ApplicationActivity.InitializeComponent().

Reply:
You are right, updated